spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -38,7 +38,7 @@
       date_created: ["2021-1-1"],
       mime_type: 'application/pdf',
       grants_and_funding: ["European Commission 00k4n6c32"],
-      language: ["English"],
+      language: ["Pali", "English", "Fake Language"],
```

Comment:
Simple test case to cover:
- English 
- Non-English and supported
- Not supported

Reply:
Is there a value that may not be supported?